packages/onboarding/src/flow-progress/index.tsx

```diff
@@ -0,0 +1,26 @@
+import './style.scss';
+
+interface Props {
+	flowLength: number;
+	positionInFlow: number;
```

Comment:
I would suggest a rename here:
`positionInFlow` => `progress`.

The reason is that this count is probably unrelated to the concept of `flow` in the stepper. For example, in the Woo part of the site setup flow the progress component only appears in the addressStore step.